use error-chain::error_chain;
use std::io::Read;

error_chain

fn main() -> Result
